MdeModulePkg/Universal/FaultTolerantWriteDxe/FtwMisc.c

```diff
@@ -810,12 +810,18 @@ FtwGetLastWriteHeader (
   FtwHeader       = (EFI_FAULT_TOLERANT_WRITE_HEADER *)(FtwWorkSpaceHeader + 1);
   Offset          = sizeof (EFI_FAULT_TOLERANT_WORKING_BLOCK_HEADER);
 
+  if (!CompareGuid (&FtwWorkSpaceHeader->Signature, &gEdkiiWorkingBlockSignatureGuid)) {

```

Comment:
Github does not let you provide review feedback on commit messages, so the comment goes here.

Please follow the commit message format as documented in https://github.com/tianocore/tianocore.github.io/wiki/Commit-Message-Format

Reply:
@leiflindholm 

 I have changed the commit message and cross checked it with `PatchCheck.py`.
can you please check again now